concordium-consensus/src/Concordium/External.hs

```diff
@@ -312,6 +312,8 @@ startConsensus ::
     Word64 ->
     -- |Number of seconds between transaction table purging runs
     Word64 ->
+    -- |Block state flush mode
+    Word64 ->
```

Comment:
It would be useful to document what the meaning of the number is, e.g., 0 for *, 1 for *.